pyquest/drawer.py

```diff
@@ -1,17 +1,17 @@
-'''
```

Comment:
A lot of these changes seem to be the result of a linter and make inconsequential, stylistic changes. This isn't really nice to do to someone's existing code; it muddies the blame, and CI can auto-lint if it is crucial to a project. PyQuEST does not even have a style guide, so the many of these changes (e.g. changing single-character `'x'` to `"x"` - hell, that's not even in [PEP 8](https://peps.python.org/pep-0008/#string-quotes)) are completely subjective and a bit pointless